moonraker/components/notifier.py

```diff
@@ -53,6 +54,21 @@ def __init__(self, config: ConfigHelper) -> None:
                 continue
             self.notifiers[notifier.get_name()] = notifier
 
+    def register_remote_actions(self):
+        self.server.register_remote_method("notify", self.notify_action)
+
+    async def notify_action(self, name: str, body: str = None):
+        if name not in self.notifiers:
+            raise self.server.error(f"Notifier '{name}' not found", 404)
+        notifier = self.notifiers[name]
+
+        kapis: APIComp = self.server.lookup_component('klippy_apis')
+        result: Dict[str, Any] = await kapis.query_objects(
+            {'print_stats': None}, default={})
+        print_stats = result.get('print_stats', {})
+
+        await notifier.notify("remote_action", [print_stats, print_stats], body)
```

Comment:
I don't think we need to include print_stats here.  If Klipper is going to define the body that is what should be sent.  No need to include print stats in the jinja context since its not triggered by a print event.

Reply:
Good point. What do you think would be a better fit to pass through?

Reply:
Sorry for the delay, life getting a bit in the way.  

I think the remote method could take a "message" argument and we could pass it through to the context.  So for example:
```python
async def notify_action(self, name: str, message: str = ""):
    if name not in self.notifiers:
        raise self.server.error(f"Notifier '{name}' not found", 404)
    notifier = self.notifiers[name]
    await notifier.notify("remote_action", [], message)
```

Then in the `NotifierInstance` class
```python
async def notify(
        self, event_name: str, event_args: List, message: str = ""
) -> None:
    context = {
        "event_name": event_name,
        "event_args": event_args,
        "event_message": message
    }
```

The user could then render the `event_message` in the template.

Reply:
No problem! No need to hurry. Good feedback! I updated the code accordingly.